addons/OPT/SHOP/fn_clientInit.sqf

```diff
@@ -0,0 +1,356 @@
+/**
+* Description:
+* 
+* 
+* Author:
+* [GNC]Lord-MDB
+*
+* Arguments:
+* 
+*
+* Return value:
+* 
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* 
+*/
+
+#include "macros.hpp"
+
+DUMP("Successfully loaded the OPT/Shop module on the client");
+
+// Events
+
+[
+	EVENT_SHOP_KAUF_ONLOAD, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call FUNC(einlesenInShopDialog);
+	},
+	[]
+	
+] call CFUNC(addEventHandler);
+
+[
+	EVENT_SHOP_KONFIG_ONLOAD, 
+	{
+		_this params ["_eventArgs"];
+
+		_eventArgs call FUNC(einlesenInKonfigDialog);
+
+	},
+	[]
+	
+] call CFUNC(addEventHandler);
+
+// Entfernung des Boxchecks PFH Shop
+[{
+	if (isNull (findDisplay 20000 displayCtrl 20002)) then 
+	{
+		GVAR(idPadCheckShop) call CFUNC(removePerframeHandler);	
+	};
+
+}, 1] call CFUNC(addPerFrameHandler);
+
+// Entfernung des Boxchecks PFH Konfig
+[{
+	if (isNull (findDisplay 22000 displayCtrl 20002)) then 
+	{
+		GVAR(idPadCheckKonfig) call CFUNC(removePerframeHandler);	
+	};
+
+}, 1] call CFUNC(addPerFrameHandler);
+
+//Shop Dialog öffnen
+if (hasInterface) then 
+{
+
+    /*
+    * https://cbateam.github.io/CBA_A3/docs/files/keybinding/fnc_addKeybind-sqf.html
+    */
+    [
+        "OPT", 
+        "OPT Shop System", 
+        ["Shop-Dialog öffnen", "Öffnet den Shop-Dialog im Fahnenbereich."], 
+        {
+            private _triggerUnitsair = 
+			(list csat_trigger_Shop_Air1) 
+			+ (list nato_trigger_Shop_Air1);
+
+			private _triggerUnitsveh = 
+			(list csat_trigger_Shop_veh1) 
+			+ (list nato_trigger_Shop_veh1);
+
+			private _triggerUnitssup = 
+			(list csat_trigger_Shop_Sup1) 
+			+ (list nato_trigger_Shop_Sup1);
+
+			private _triggerUnitssea = 
+			(list csat_trigger_Shop_Sea1) 
+			+ (list nato_trigger_Shop_Sea1);
+
+            if (player in _triggerUnitsair) then 
+			{
+                 [EVENT_SHOP_KAUF_ONLOAD,["choppers"]] call CFUNC(localEvent);
+            };
+
+			if (player in _triggerUnitsveh) then 
+			{
+                [EVENT_SHOP_KAUF_ONLOAD,["vehicles"]] call CFUNC(localEvent);
+            };
+
+			if (player in _triggerUnitssup) then 
+			{
+                [EVENT_SHOP_KAUF_ONLOAD,["supplies"]] call CFUNC(localEvent);
+            };
+
+			if (player in _triggerUnitssea) then 
+			{
+                [EVENT_SHOP_KAUF_ONLOAD,["sea"]] call CFUNC(localEvent);
+            };
+        }, 
+        {}, 
+        [
+            DIK_F3, 
+            [false, false, false] // [shift, ctrl, alt]
+        ]
+    ] call CBA_fnc_addKeybind;
+
+};
+
+//Verkaufs Dialog öffnen
+if (hasInterface) then 
+{
```

Comment:
Der folgende Block könnte einfach in den `hasInterface`-Block darüber mit integriert werden. Dann muss man das nicht mehrfach checken.

Das gilt auch für alle folgenden `hasInterface` checks.